src/main/scala/cromwell/Main.scala

```diff
@@ -3,50 +3,104 @@ package cromwell
 import java.io.File
 
 import cromwell.binding._
+import cromwell.engine.SingleWorkflowRunner
 import cromwell.parser.WdlParser.SyntaxError
 import cromwell.server.CromwellServer
-
+import spray.json._
+import scala.util.{Failure, Success}
 
 object Actions extends Enumeration {
-  val parse, validate, run, server = Value
+  val parse, validate, run, inputs, server = Value
```

Comment:
By convention these should probably have an initial capital letter.  If we want to keep the CLI the same we can do:

```
getAction(args.headOption map { _.capitalize }) match {
```